proxy/logging/LogFieldAliasMap.h

```diff
@@ -130,7 +136,7 @@ class LogFieldAliasTable : public LogFieldAliasMap
   void init(size_t numPairs, ...);
 
   int
-  asInt(char *key, IntType *val, bool case_sensitive) const override
+  asInt(std::string_view key, IntType *val, bool case_sensitive) const override
```

Comment:
What enum/type is INVALID_STRING and ALL_OK? What I meant here was, since you are mucking around, why not change the return to be of that type? A function named asInt() which returns an Int, which is really an enum with status codes, is really confusing.